opentelemetry-sdk/tests/metrics/prototype/http_server/test_http_server.py

```diff
@@ -0,0 +1,178 @@
+from asyncio import Queue, gather, sleep
+from random import choice, randint, random, seed
+from sys import version_info
+from unittest import TestCase
+
+from pytest import mark
+
+from opentelemetry.sdk.metrics.aggregator import LastAggregator
+from opentelemetry.sdk.metrics.meter import MeterProvider
+
+seed(3)
+
+
+async def main():
+    addresses_queues = {
+        32: Queue(),
+        33: Queue(),
+        34: Queue(),
+        35: Queue(),
+        36: Queue(),
+        37: Queue(),
+    }
+
+    meter = MeterProvider().get_meter("meter")
+
+    active_requests = meter.create_up_down_counter(
+        "active_requests",
+        unit="active requests",
+        description="Currently active requests",
+    )
+
+    humidity = meter.create_counter(
+        "humidity",
+        unit="percentage",
+        description="Server humidity",
+        aggregator_class=LastAggregator,
+    )
+
+    temperature = meter.create_up_down_counter(
+        "temperature",
+        unit="F",
+        description="Server temperature",
+        aggregator_class=LastAggregator,
+    )
+
+    tester_queue = Queue()
+
+    class Push:
+        async def start(self):
+
+            while True:
+                await sleep(1)
+                self.export()
+                await tester_queue.put(
+                    active_requests.value(host_name="MachineA")
+                )
+
+    class Exporter:
```

Comment:
Can this be replaced by the `ConsoleExporter`? I'm not sure I understand why the export method here is different than the exporter interface.